exercises/go-counting/canonical-data.json

```diff
@@ -21,7 +19,11 @@
       },
       "expected": {
         "owner": "BLACK",
-        "territory": [[0, 0], [0, 1], [1, 0]]
+        "territory": [
+          [0, 0],
+          [0, 1],
+          [1, 0]
+        ]
```

Comment:
I'm not sure if this is necessarily better than the single-line versions but I'm not objecting to it either.